packages/controller-utils/src/create-service-policy.ts

```diff
@@ -130,6 +131,18 @@ export const DEFAULT_CIRCUIT_BREAK_DURATION = 30 * 60 * 1000;
  */
 export const DEFAULT_DEGRADED_THRESHOLD = 5_000;
 
+const isServiceFailure = (error: unknown) => {
```

Comment:
What about HTTP 4XX responses? We can exclude those as being indicative of service failure as well

Reply:
Ugh. I see why - we wrap all HTTP errors in an "internal error". I forgot we did that, that is problematic.

As much as I'd like to improve our error handling separately, I think we at least need to split 5XX from 4XX here in order to make the circuit breaker work correctly.